### DISCUSSION TITLE
Technical Bootcamp Retrospective - Please answer the following questions

### DISCUSSION THREAD

**Discussion by davidgitit** [Drook-tech/Bootcamp#12] [Q&A]
What do we do well?

What could we have done better?

Ideas and suggestions?

**Comment by william-liebenberg** [comment]
What do we do well?
- 2 days Technical bootcamp covered a lot of features.
- the timing for each exercise was just right

What could we have done better?
- can't think of anything

Ideas and suggestions?
- Probably need a whole bootcamp for CodeQL :)

LYW! thanks

**Comment by Saraquin** [comment]
**What do we do well?**

- Depth of detail
- Topic flow

**What could we have done better?**

- Notes on gotcha's in the doc we followed, a few items were only mentioned in the video and I forget one which stumped me.

**Ideas and suggestions?**

- Timeline on the video for topic reference
- A bit slower on the lab walk through
- Distribute a jargon buster for us with out a Dev background

Cheers
